R/utils.R

```diff
@@ -22,3 +22,34 @@ description_file_attribute <- function(pkg_path, attribute) {
   as.character(read.dcf(file.path(pkg_path, "DESCRIPTION"))[1, attribute])
 }
 
+`commit_to_gh_pages!` <- function(directory, dir) {
+  ## A little bit of git magic
+
+  ## Now that we have the docs, we need to transfer them to the gh-pages
+  ## branch of the repo, commit and push.
+  ## Switch to gh-pages branch
```

Comment:
I think they're better about all the edge cases that can happen in your git status.